ERC20ForSPL/test/ERC20ForSPLMintable.js

```diff
@@ -57,14 +57,18 @@ describe('Test init', async function () {
         let tx = await ERC20ForSPLMintableFactory.deploy(
             'Testcoin',
             'TST',
+            'https://pbs.twimg.com/profile_images/1708826035695853568/TxbDTDsu_400x400.jpg',
```

Comment:
According to uri, it should follow a certain standard. You can find its description here: https://docs.metaplex.com/programs/token-metadata/token-standard#the-fungible-standard.

This follows from `Create Metadata Account Args V3` instruction description (https://docs.metaplex.com/programs/token-metadata/instructions#create-a-metadata-account)

> The URI of the token, limited to 200 bytes. This URI points to an off-chain JSON file that contains additional data following a certain standard. You can learn more about this JSON standard [here](https://docs.metaplex.com/programs/token-metadata/token-standard).The JSON file can either be stored in a traditional server (e.g. using AWS) or using a permanent storage solution such as using Arweave.

Many of tokens has empty `uri` field. I found some tokens as example:
- https://solscan.io/token/HZ1JovNiVvGrGNiiYvEozEVgZ58xaU3RKwX8eACQBCt3#metadata
- https://solscan.io/token/5MAYDfq5yxtudAhtfyuMBuHZjgAbaS9tbEyEQYAhDS5y#metadata